### ISSUE TITLE
feature req.

### ISSUE THREAD

**Issue by Lidocian** [ike3/mangosbot-zero#7]
Hi Ike3!
I want to thank you for your hard work with the playerbots especialy mangosbot-zero version!

some feature which  are needed or make your bots better i think :

1. **Battleground**
       WSG is working!! with your bots on trinitycore 3.3.5 with conan513@699042d. 
        (http://www.ac-web.org/forums/showthread.php?221383-Single-Player-Project-TC-R2)

2. **whisper** 
       please add a function to turn on/off the bots whispering you at all ( gather, trade, quest etc..)

3. **enchant** 
       enchant bot  equipment when bot created/initalized.

4.  **talent template** 
         editable for bots  to create 2-3 build for each class spec and set via conf.

5. **speech** 
       not just at taunt,low hp/mana/aoe(i like it ) but randomly

**Comment by wrathix** [comment]
Yes, thanks for continuing with the bots still , always fantastic.   I look at your commits daily.
Please dont give up on the trinitycore 3.3.5a version though :)

Merry xmas
